Sources/TokamakDemo/ShapeStyleDemo.swift

```diff
@@ -92,14 +92,44 @@ struct ShapeStyleDemo: View {
     HStack {
       VStack {
         Text("Primary")
-          .foregroundStyle(PrimaryContentStyle())
+          .foregroundStyle(HierarchicalShapeStyle.primary)
         Text("Secondary")
-          .foregroundStyle(SecondaryContentStyle())
+          .foregroundStyle(HierarchicalShapeStyle.secondary)
         Text("Tertiary")
-          .foregroundStyle(TertiaryContentStyle())
+          .foregroundStyle(HierarchicalShapeStyle.tertiary)
         Text("Quaternary")
-          .foregroundStyle(QuaternaryContentStyle())
+          .foregroundStyle(HierarchicalShapeStyle.quaternary)
       }
+      VStack {
+        Text("Primary")
+          .foregroundStyle(HierarchicalShapeStyle.primary)
+        Text("Secondary")
+          .foregroundStyle(HierarchicalShapeStyle.secondary)
+        Text("Tertiary")
+          .foregroundStyle(HierarchicalShapeStyle.tertiary)
+        Text("Quaternary")
+          .foregroundStyle(HierarchicalShapeStyle.quaternary)
```

Comment:
SwiftUI supports the following shorter form syntax, so we should as well (and use it in some of the demo).
```suggestion
        Text("Primary")
          .foregroundStyle(.primary)
        Text("Secondary")
          .foregroundStyle(.secondary)
        Text("Tertiary")
          .foregroundStyle(.tertiary)
        Text("Quaternary")
          .foregroundStyle(.quaternary)
```

Reply:
Correct me if I’m wrong but I believe that requires Swift 5.5. The extensions on ShapeStyle to allow it are there, but for now we’ll have to be explicit about the type name. https://github.com/apple/swift-evolution/blob/main/proposals/0299-extend-generic-static-member-lookup.md

Reply:
Yes, I had the same impression that this requires 5.5, and we haven't even got SwiftWasm 5.4 ready for Tokamak yet...

Reply:
Sure, but we can get the same functionality in other ways. It just takes a bit more code.

Here's what I'm thinking: ezraberch/Tokamak@9ea989d0c551ed07e4201a7f8501a355ed76d1c3

Reply:
If we did work around that, it may be better to add something more general like the original SwiftUI beta's `StaticMember<Base>` struct. At the same time, I wonder if that would overlap with the `ShapeStyle` extensions which should currently work with `TokamakStaticHTML`/`TokamakGTK` in Swift 5.5.

Reply:
Either way, I'm totally fine with merging it as it is. It would be easier to discuss workarounds in a separate PR/issue, or maybe we'll get 5.5 working by that point.